src/dbnode/namespace/types.go

```diff
@@ -221,6 +221,11 @@ type Registry interface {
 	// Watch for the Registry changes
 	Watch() (Watch, error)
 
+	// ForceGet bypasses the watch and issues a get request against KV directly.
```

Comment:
nit: “against the backing store.”

The interface has the cfg implementation too